compiler/rustc_type_ir/src/const_kind.rs

```diff
@@ -13,7 +11,15 @@ use crate::{
 use self::ConstKind::*;
 
 /// Represents a constant in Rust.
-// #[derive(derive_more::From)]
+#[derive(derivative::Derivative)]
+#[derivative(
+    Clone(bound = ""),
+    PartialOrd(bound = ""),
+    PartialOrd = "feature_allow_slow_enum",
```

Comment:
ironically "feature_allow_slow_enum" wasn't even the cause for the slowness: https://github.com/rust-lang/rust/pull/117409#issuecomment-1786263161